### PR TITLE
Pushing my fork change upstream

### PR DESCRIPTION
**Pull request by meteatamel**

### CODE REVIEW

File: web/catrelay.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 const nconf = require('nconf');
 const util = require('util');
 const gcloud = require('gcloud')({
-  projectId: 'cloudcats-next',
+  projectId: process.env.GCLOUD_PROJECT,
```
Comment (stephenplusplus) [inline web/catrelay.js]: `gcloud` will look for this env var by default, so no need to specify here unless you think being explicit is better for this demo.

Comment (meteatamel) [inline web/catrelay.js]: Ok, that's good to know.

Comment (JustinBeckwith) [inline web/catrelay.js]: Could you remove this and make another commit against the PR?

File: web/package.json [hunk 2]
```diff
@@ -13,8 +13,8 @@
     "node": "^4.4.0"
   },
   "dependencies": {
-    "@google/cloud-debug": "^0.6.2",
-    "@google/cloud-trace": "github:googlecloudplatform/cloud-trace-nodejs#canary",
+    "@google/cloud-debug": "^0.8.1",
+    "@google/cloud-trace": "^0.5.1",
```
Comment (JustinBeckwith) [inline web/package.json]: I think this PR is kinda out of date.  Would you mind pulling from master?

**Comment by JustinBeckwith** [comment]
Sorry, this PR got moldy :(
